src/shipit_code_coverage/shipit_code_coverage/notifier.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+from urllib.error import HTTPError
+import requests
+
+from cli_common.log import get_logger
+from cli_common.taskcluster import get_service
+
+from shipit_code_coverage.utils import wait_until
+
+
+logger = get_logger(__name__)
+
+
+class Notifier(object):
+
+    def __init__(self, revision, emails, client_id, access_token):
+        self.revision = revision
+        self.emails = emails
+        self.notify_service = get_service('notify', client_id, access_token)
+
+    def get_coverage_summary(self, changeset):
+        r = requests.get('https://uplift.shipit.staging.mozilla-releng.net/coverage/changeset_summary/%s' % changeset)
```

Comment:
You should use a TC secret here, as code coverage in prod should use the uplift server from prod

Reply:
Makes sense, I'll do that.

Reply:
Actually, let's do it in a follow-up, since this code was actually pre-existing I'd like to have a separate PR implementing it.